update/src/main.rs

```diff
@@ -6,7 +6,7 @@ use syn::{ForeignItem, Ident, Item};
 fn main() {
     const USER: &str = "bitcoin-core";
     const REPO_NAME: &str = "secp256k1";
-    const COMMIT_SHA: &str = "2bca0a5cbf756dd4ff1f0bda4585a7d3c64e1480";
+    const COMMIT_SHA: &str = "c55290c0ae1031dcb8d8fed4fe3d6ab3f22c0511";
```

Comment:
@sergey-shandar does updating this `COMMIT_SHA` mean we need to re-run update?

Reply:
I misunderstood the purpose of this sha and set it to this repo's previous commit. Reverted in latest commit.

Reply:
I think, I'm late for the discussion, but yes, you need to re-rurn `cargo update`. It's in the README file https://github.com/Trust-Machines/p256k1/tree/master/update ;-)